docs/pages/router/static-rendering/async-routes.mdx

```diff
@@ -25,27 +26,29 @@ Async routes cannot be statically filtered during development, so all files will
 
 ## Static rendering
 
-The current static rendering system doesn't support React Suspense, so we have to disable it when rendering static pages. This will be improved in the future.
+The current static rendering system (React) doesn't support React Suspense, so async routes will be disabled on web when rendering static pages. This will be improved in the future.
 
 ## Setup
 
-Configure Metro to use Expo's bundle splitting features:
-
-```js metro.config.js
-const { getDefaultConfig } = require('@expo/metro-config');
-const config = getDefaultConfig(__dirname);
-config.transformer = {
-  ...config.transformer,
-  asyncRequireModulePath: require.resolve('@expo/metro-runtime/async-require'),
-};
-config.server = {
-  ...config.server,
-  experimentalImportBundleSupport: true,
-};
-module.exports = config;
+Enable the feature by setting the `asyncRoutes` option to `development` in your **app.json**:
```

Comment:
```suggestion
Enable the feature by setting the `asyncRoutes` option to `development` in your [app config](/versions/latest/config/app/):
```

@EvanBacon, do we add this property in [app config schema](https://docs.expo.dev/versions/latest/config/app/)?